junit5-gradle-consumer/build.gradle

```diff
@@ -34,43 +24,40 @@ compileTestJava {
 	options.compilerArgs += '-parameters'
 }
 
-junitPlatform {
-	// platformVersion '1.1.0'
-	filters {
-		engines {
-			// include 'junit-jupiter', 'junit-vintage'
-			// exclude 'custom-engine'
-		}
-		tags {
-			// include 'fast'
-			exclude 'slow'
-		}
-		// includeClassNamePattern '.*Test'
+test {
+	useJUnitPlatform {
+		// includeEngines 'junit-jupiter', 'junit-vintage'
+		// excludeEngines 'custom-engine'
+
+		// includeTags 'fast'
+		excludeTags 'slow'
+	}
+
+	testLogging {
+		events 'passed', 'skipped', 'failed'
+	}
+
+	reports {
+		html.enabled = true
 	}
-	// configurationParameter 'junit.jupiter.conditions.deactivate', '*'
-	// enableStandardTestTask true
-	// reportsDir file('build/test-results/junit-platform') // this is the default
-	logManager 'org.apache.logging.log4j.jul.LogManager'
```

Comment:
Why is this no longer needed?

I used to replace it with: `systemProperty 'java.util.logging.manager', 'org.apache.logging.log4j.jul.LogManager'`

Reply:
Ah, I'm sorry about this omission. Should I add `systemProperty 'java.util.logging.manager', 'org.apache.logging.log4j.jul.LogManager'` back to the `test {}` configuration?

Reply:
No. It's okay as it is now. The configuration now looks cleaner and there's no "verbose" garbage at the output. Deleted the log4j xml file, too.